### ISSUE TITLE
Support F# in .NET Core

### ISSUE THREAD

**Issue by ashmind** [ashmind/SharpLab#407]
Requires ashmind/mirrorsharp#97 which should be unblocked now.

**Comment by baronfel** [comment]
Related to this, are there other features that lack f# support that I could help close the gap on?

**Comment by ashmind** [comment]
@baronfel
Yep, but I am still trying to resolve few issues in this one, so might be worth waiting a bit.
From what I remember:
1. Quick Info ashmind/mirrorsharp#89
2. Signature Help ashmind/mirrorsharp#66
3. Flow Visualization https://twitter.com/ashmind/status/894068736117809153

**Comment by ashmind** [comment]
Sadly I am having significant performance issues when running F# in Profiler mode.
I can't diagnose them since on local it even crashes consistently.

I'll keep basic .NET Core support here, but I'll split Profiler mode support into a separate ticket.

**Comment by ashmind** [comment]
F# is now available on .NET Core, but Profiled mode will be handled separately in #409.
Any performance improvements for F# in mirrorsharp would be much appreciated.
